locale/en_US/default.po

```diff
@@ -140,5 +140,7 @@ msgstr "R"
 msgid "default.contextSettings.emailSignature"
 msgstr ""
 "<br/>\n"
-"________________________________________________________________________<br/>\n"
-"<a href=\"{$contextUrl}\">{$contextName}</a>"
+"<hr /><br/>\n"
+"<p class='emaiSignatureContextUrlStyle'><a href=\"{$contextUrl}\">{$contextName}</a></p>"
```

Comment:
I think `contextName` needs to be escaped.

Reply:
Could I escape that here, like `{$contextUrl|escape}`? Or should I do that in the when the variable is created?

Reply:
Because we're not processing these as Smarty templates, `|escape` wouldn't work here. It should be done when the variable is replaced into the default text.

Reply:
Perhaps something like 

```
'mailingAddress' => addslashes($this->context->getData('mailingAddress')),
'contactEmail' => addslashes($this->context->getData('contactEmail')),
'contactName' => addslashes($this->context->getData('contactName')),
```

at https://github.com/defstat/pkp-lib/blob/i5049/classes/mail/MailTemplate.inc.php#L153-L155

Reply:
Since these variables are going into the body, it should be `htmlspecialchars` rather than `addslashes`. (Plus as above `mailingAddress` needs to be turned from plaintext into HTML, so that one should have line breaks turned into `<br/>`.)